blueocean-commons/src/main/java/io/jenkins/blueocean/commons/stapler/export/Property.java

```diff
@@ -141,15 +141,14 @@ public void writeTo(Object object, TreePruner pruner, DataWriter writer) throws
         if (merge) {
             // merged property will get all its properties written here
             if (d != null) {
-                Model model;
-                try {
-                    model = owner.get(d.getClass(), parent.type, name);
-                } catch (NotExportableException e) {
+                if (d.getClass().getAnnotation(ExportedBean.class) == null) {
```

Comment:
Changes in Property.java in this PR is different from the ones in PR. I think its cleaner to remove changes from here and merge stapler changes in separate PR, after stapler PR is merged. This way we can merge this PR without waiting for stapler changes and get immediate perf benefits.